src/api/create-env-test-suite/controller.js

```diff
@@ -24,57 +18,20 @@ const createEnvTestSuiteController = {
     }
   },
   handler: async (request, h) => {
-    const gitHubOrg = config.get('gitHubOrg')
-
     const payload = request?.payload
     const repositoryName = payload?.repositoryName
+    const user = request.auth?.credentials
 
-    const zone = 'public'
-    const { team } = await fetchTeam(payload?.teamId)
-    if (!team?.github) {
-      throw Boom.badData(`Team ${team.name} does not have a linked Github team`)
-    }
-
-    request.logger.info(`Creating env test suite: ${repositoryName}`)
-
-    try {
-      await createTestSuiteStatus(
-        request.db,
-        gitHubOrg,
-        repositoryName,
-        zone,
-        team,
-        creations.envTestsuite,
-        'cdp-node-env-test-suite-template'
-      )
-    } catch (e) {
-      request.logger.error(e)
-      throw Boom.badData(
-        `repository ${repositoryName} has already been requested or is in progress`
-      )
-    }
-
-    const template = config.get('createEnvTestSuiteWorkflow')
-    await createTestSuiteFromTemplate(
+    await createTestRunnerSuite(
```

Comment:
When there starts to be a lot of args to methods, anything `> 4` I usually switch to use an object. It makes things easier to understand and means that things are called the same thing in the args and the method body. Also is super handy for supplying defaults args

```
await createTestRunnerSuite({ 
  request, 
  repositoryName, 
  envTestsuite: creations.envTestsuite,
  teamId: payload?.teamId,
  user,
  workFlowsCreateEnvTestSuite config.get('workflows.createEnvTestSuite'),
  environments: ['environment']}
})
```